src/script/conversation/ConversationState.ts

```diff
@@ -137,6 +137,14 @@ export class ConversationState {
     return proteusConversation;
   }
 
+  getSelfMLSConversation(): Conversation {
+    const proteusConversation = this.selfMLSConversation();
+    if (!proteusConversation) {
```

Comment:
proteusConversation won't be undefined right?

Reply:
We should rename it `proteusConversation` to `mlsConversation` I guess it's copy-paste mistake here 😅

Reply:
It can if we are in a non-mls environment

Reply:
@PatrykBuniX oh damn 😅

Reply:
😆

Reply:
in that case it should be !! mlsConversation?